### PR TITLE
Profil Page With Responsive

### PR DESCRIPTION
**Pull request by pagouillart**

### CODE REVIEW

**Review by Rayane0823 — APPROVED** [review]

**Review by Cyrildab — APPROVED** [review]

**Review by Romlion — APPROVED** [review]

**Review by LucasMerino10 — APPROVED** [review]

File: frontend/src/components/Profil/ProfilCard.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import ProfilPic from "../../assets/Profil.png";
+import "./ProfilCard.scss";
+
+function ProfilCard(userProfil) {
+  const { name, lastname, email, password } = userProfil;
+  return (
+    <section className="profilCard">
```
Comment (Yoann-Fortin) [inline frontend/src/components/Profil/ProfilCard.jsx]: Reversal of required tags

File: frontend/src/components/Profil/ProfilCard.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <img src={ProfilPic} alt="Profil" className="profilPic" />
+      <article className="profilInfo">
```
Comment (Yoann-Fortin) [inline frontend/src/components/Profil/ProfilCard.jsx]: Reversal of required tags

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/Profil/ProfilCard.jsx (hunk 1 above)
Comment (pagouillart) [reply frontend/src/components/Profil/ProfilCard.jsx]: done

File: frontend/src/components/Profil/ProfilCard.jsx (hunk 2 above)
Comment (pagouillart) [reply frontend/src/components/Profil/ProfilCard.jsx]: done

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**